src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceSuggestionExamples.tsx

```diff
@@ -11,76 +12,45 @@ type Props = {
   sourceName: string;
 };
 
-type State = {
-  isOpen: boolean;
-};
-
-class SourceSuggestionExamples extends React.Component<Props, State> {
-  state: State = {isOpen: false};
-
-  toggleModal = () => {
-    this.setState({isOpen: !this.state.isOpen});
-  };
-
-  stopPropagation = (e: React.MouseEvent<HTMLSpanElement>) => {
-    // Necessary to stop propagation of click events from modal that we can't
-    // catch otherwise.
-    e.stopPropagation();
-  };
-
-  render() {
-    const {isOpen} = this.state;
-    const {examples, sourceName} = this.props;
-
-    return (
-      <Wrapper onClick={this.stopPropagation}>
-        <Button size="xsmall" onClick={this.toggleModal}>
-          {t('examples')}
-        </Button>
-        {isOpen && (
-          <StyledModal show onHide={this.toggleModal}>
-            <Modal.Header closeButton>
-              {t('Examples for %s in current event', <code>{sourceName}</code>)}
-            </Modal.Header>
-            <Modal.Body>
-              {examples.map(example => (
-                <pre key={example}>{example}</pre>
-              ))}
-            </Modal.Body>
-          </StyledModal>
-        )}
-      </Wrapper>
-    );
-  }
-}
+const SourceSuggestionExamples = ({examples, sourceName}: Props) => (
```

Comment:
```suggestion
function SourceSuggestionExamples({examples, sourceName}: Props) {
```

maybe? 🤷‍♀️

Reply:
I'm indifferent tbh, it would be nice if we could add some linitng to prefer one way or the other for top level stuff.